Description: no boringssl source
 As boringssl is not packaged, can't use it.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2026-07-05

---

--- a/src/python/grpcio/grpc_core_dependencies.py
+++ b/src/python/grpcio/grpc_core_dependencies.py
@@ -963,287 +963,6 @@ CORE_SOURCE_FILES = [
     'third_party/address_sorting/address_sorting.c',
     'third_party/address_sorting/address_sorting_posix.c',
     'third_party/address_sorting/address_sorting_windows.c',
-    'third_party/boringssl-with-bazel/crypto/aes/aes.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_bitstr.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_bool.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_d2i_fp.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_dup.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_gentm.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_i2d_fp.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_int.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_mbstr.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_object.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_octet.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_strex.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_strnid.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_time.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_type.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/a_utctm.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/asn1_lib.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/asn1_par.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/asn_pack.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/f_int.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/f_string.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/posix_time.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/tasn_dec.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/tasn_enc.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/tasn_fre.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/tasn_new.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/tasn_typ.cc',
-    'third_party/boringssl-with-bazel/crypto/asn1/tasn_utl.cc',
-    'third_party/boringssl-with-bazel/crypto/base64/base64.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/bio.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/bio_mem.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/connect.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/errno.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/fd.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/file.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/hexdump.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/pair.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/printf.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/socket.cc',
-    'third_party/boringssl-with-bazel/crypto/bio/socket_helper.cc',
-    'third_party/boringssl-with-bazel/crypto/blake2/blake2.cc',
-    'third_party/boringssl-with-bazel/crypto/bn/bn_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/bn/convert.cc',
-    'third_party/boringssl-with-bazel/crypto/bn/exponentiation.cc',
-    'third_party/boringssl-with-bazel/crypto/buf/buf.cc',
-    'third_party/boringssl-with-bazel/crypto/bytestring/asn1_compat.cc',
-    'third_party/boringssl-with-bazel/crypto/bytestring/ber.cc',
-    'third_party/boringssl-with-bazel/crypto/bytestring/cbb.cc',
-    'third_party/boringssl-with-bazel/crypto/bytestring/cbs.cc',
-    'third_party/boringssl-with-bazel/crypto/bytestring/unicode.cc',
-    'third_party/boringssl-with-bazel/crypto/chacha/chacha.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/derive_key.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_aesctrhmac.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_aeseax.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_aesgcmsiv.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_chacha20poly1305.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_des.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_null.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_rc2.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_rc4.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/e_tls.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/get_cipher.cc',
-    'third_party/boringssl-with-bazel/crypto/cipher/tls_cbc.cc',
-    'third_party/boringssl-with-bazel/crypto/cms/cms.cc',
-    'third_party/boringssl-with-bazel/crypto/conf/conf.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_aarch64_apple.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_aarch64_fuchsia.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_aarch64_linux.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_aarch64_openbsd.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_aarch64_sysreg.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_aarch64_win.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_arm_freebsd.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_arm_linux.cc',
-    'third_party/boringssl-with-bazel/crypto/cpu_intel.cc',
-    'third_party/boringssl-with-bazel/crypto/crypto.cc',
-    'third_party/boringssl-with-bazel/crypto/curve25519/curve25519.cc',
-    'third_party/boringssl-with-bazel/crypto/curve25519/curve25519_64_adx.cc',
-    'third_party/boringssl-with-bazel/crypto/curve25519/spake25519.cc',
-    'third_party/boringssl-with-bazel/crypto/des/des.cc',
-    'third_party/boringssl-with-bazel/crypto/dh/dh_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/dh/params.cc',
-    'third_party/boringssl-with-bazel/crypto/digest/digest_extra.cc',
-    'third_party/boringssl-with-bazel/crypto/dsa/dsa.cc',
-    'third_party/boringssl-with-bazel/crypto/dsa/dsa_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/ec/ec_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/ec/ec_derive.cc',
-    'third_party/boringssl-with-bazel/crypto/ec/hash_to_curve.cc',
-    'third_party/boringssl-with-bazel/crypto/ecdh/ecdh.cc',
-    'third_party/boringssl-with-bazel/crypto/ecdsa/ecdsa_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/engine/engine.cc',
-    'third_party/boringssl-with-bazel/crypto/err/err.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/evp.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/evp_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/evp_ctx.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_dh.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_dh_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_dsa_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_ec.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_ec_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_ed25519.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_ed25519_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_hkdf.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_rsa.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_rsa_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_x25519.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/p_x25519_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/pbkdf.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/print.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/scrypt.cc',
-    'third_party/boringssl-with-bazel/crypto/evp/sign.cc',
-    'third_party/boringssl-with-bazel/crypto/ex_data.cc',
-    'third_party/boringssl-with-bazel/crypto/fipsmodule/bcm.cc',
-    'third_party/boringssl-with-bazel/crypto/fipsmodule/fips_shared_support.cc',
-    'third_party/boringssl-with-bazel/crypto/fuzzer_mode.cc',
-    'third_party/boringssl-with-bazel/crypto/hpke/hpke.cc',
-    'third_party/boringssl-with-bazel/crypto/hrss/hrss.cc',
-    'third_party/boringssl-with-bazel/crypto/kyber/kyber.cc',
-    'third_party/boringssl-with-bazel/crypto/lhash/lhash.cc',
-    'third_party/boringssl-with-bazel/crypto/md4/md4.cc',
-    'third_party/boringssl-with-bazel/crypto/md5/md5.cc',
-    'third_party/boringssl-with-bazel/crypto/mem.cc',
-    'third_party/boringssl-with-bazel/crypto/mldsa/mldsa.cc',
-    'third_party/boringssl-with-bazel/crypto/mlkem/mlkem.cc',
-    'third_party/boringssl-with-bazel/crypto/obj/obj.cc',
-    'third_party/boringssl-with-bazel/crypto/obj/obj_xref.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_all.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_info.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_lib.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_oth.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_pk8.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_pkey.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_x509.cc',
-    'third_party/boringssl-with-bazel/crypto/pem/pem_xaux.cc',
-    'third_party/boringssl-with-bazel/crypto/pkcs7/pkcs7.cc',
-    'third_party/boringssl-with-bazel/crypto/pkcs7/pkcs7_x509.cc',
-    'third_party/boringssl-with-bazel/crypto/pkcs8/p5_pbev2.cc',
-    'third_party/boringssl-with-bazel/crypto/pkcs8/pkcs8.cc',
-    'third_party/boringssl-with-bazel/crypto/pkcs8/pkcs8_x509.cc',
-    'third_party/boringssl-with-bazel/crypto/poly1305/poly1305.cc',
-    'third_party/boringssl-with-bazel/crypto/poly1305/poly1305_arm.cc',
-    'third_party/boringssl-with-bazel/crypto/poly1305/poly1305_vec.cc',
-    'third_party/boringssl-with-bazel/crypto/pool/pool.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/deterministic.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/fork_detect.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/forkunsafe.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/getentropy.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/ios.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/passive.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/rand.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/trusty.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/urandom.cc',
-    'third_party/boringssl-with-bazel/crypto/rand/windows.cc',
-    'third_party/boringssl-with-bazel/crypto/rc4/rc4.cc',
-    'third_party/boringssl-with-bazel/crypto/refcount.cc',
-    'third_party/boringssl-with-bazel/crypto/rsa/rsa_asn1.cc',
-    'third_party/boringssl-with-bazel/crypto/rsa/rsa_crypt.cc',
-    'third_party/boringssl-with-bazel/crypto/rsa/rsa_extra.cc',
-    'third_party/boringssl-with-bazel/crypto/rsa/rsa_print.cc',
-    'third_party/boringssl-with-bazel/crypto/sha/sha1.cc',
-    'third_party/boringssl-with-bazel/crypto/sha/sha256.cc',
-    'third_party/boringssl-with-bazel/crypto/sha/sha512.cc',
-    'third_party/boringssl-with-bazel/crypto/siphash/siphash.cc',
-    'third_party/boringssl-with-bazel/crypto/slhdsa/slhdsa.cc',
-    'third_party/boringssl-with-bazel/crypto/spake2plus/spake2plus.cc',
-    'third_party/boringssl-with-bazel/crypto/stack/stack.cc',
-    'third_party/boringssl-with-bazel/crypto/thread.cc',
-    'third_party/boringssl-with-bazel/crypto/thread_none.cc',
-    'third_party/boringssl-with-bazel/crypto/thread_pthread.cc',
-    'third_party/boringssl-with-bazel/crypto/thread_win.cc',
-    'third_party/boringssl-with-bazel/crypto/trust_token/pmbtoken.cc',
-    'third_party/boringssl-with-bazel/crypto/trust_token/trust_token.cc',
-    'third_party/boringssl-with-bazel/crypto/trust_token/voprf.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/a_digest.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/a_sign.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/a_verify.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/algorithm.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/asn1_gen.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/by_dir.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/by_file.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/i2d_pr.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/name_print.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/policy.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/rsa_pss.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/t_crl.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/t_req.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/t_x509.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/t_x509a.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_akey.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_akeya.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_alt.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_bcons.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_bitst.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_conf.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_cpols.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_crld.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_enum.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_extku.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_genn.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_ia5.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_info.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_int.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_lib.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_ncons.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_ocsp.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_pcons.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_pmaps.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_prn.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_purp.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_skey.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/v3_utl.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_att.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_cmp.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_d2.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_def.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_ext.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_lu.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_obj.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_req.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_set.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_trs.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_txt.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_v3.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_vfy.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509_vpm.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509cset.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509name.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509rset.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x509spki.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_algor.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_all.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_attrib.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_crl.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_exten.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_name.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_pubkey.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_req.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_sig.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_spki.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_val.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_x509.cc',
-    'third_party/boringssl-with-bazel/crypto/x509/x_x509a.cc',
-    'third_party/boringssl-with-bazel/gen/crypto/err_data.cc',
-    'third_party/boringssl-with-bazel/ssl/bio_ssl.cc',
-    'third_party/boringssl-with-bazel/ssl/d1_both.cc',
-    'third_party/boringssl-with-bazel/ssl/d1_lib.cc',
-    'third_party/boringssl-with-bazel/ssl/d1_pkt.cc',
-    'third_party/boringssl-with-bazel/ssl/d1_srtp.cc',
-    'third_party/boringssl-with-bazel/ssl/dtls_method.cc',
-    'third_party/boringssl-with-bazel/ssl/dtls_record.cc',
-    'third_party/boringssl-with-bazel/ssl/encrypted_client_hello.cc',
-    'third_party/boringssl-with-bazel/ssl/extensions.cc',
-    'third_party/boringssl-with-bazel/ssl/handoff.cc',
-    'third_party/boringssl-with-bazel/ssl/handshake.cc',
-    'third_party/boringssl-with-bazel/ssl/handshake_client.cc',
-    'third_party/boringssl-with-bazel/ssl/handshake_server.cc',
-    'third_party/boringssl-with-bazel/ssl/s3_both.cc',
-    'third_party/boringssl-with-bazel/ssl/s3_lib.cc',
-    'third_party/boringssl-with-bazel/ssl/s3_pkt.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_aead_ctx.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_asn1.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_buffer.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_cert.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_cipher.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_credential.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_file.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_key_share.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_lib.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_privkey.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_session.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_stat.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_transcript.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_versions.cc',
-    'third_party/boringssl-with-bazel/ssl/ssl_x509.cc',
-    'third_party/boringssl-with-bazel/ssl/t1_enc.cc',
-    'third_party/boringssl-with-bazel/ssl/tls13_both.cc',
-    'third_party/boringssl-with-bazel/ssl/tls13_client.cc',
-    'third_party/boringssl-with-bazel/ssl/tls13_enc.cc',
-    'third_party/boringssl-with-bazel/ssl/tls13_server.cc',
-    'third_party/boringssl-with-bazel/ssl/tls_method.cc',
-    'third_party/boringssl-with-bazel/ssl/tls_record.cc',
     'third_party/cares/cares/src/lib/ares_addrinfo2hostent.c',
     'third_party/cares/cares/src/lib/ares_addrinfo_localhost.c',
     'third_party/cares/cares/src/lib/ares_android.c',
